### DISCUSSION TITLE
Examples for sending requests with headers or cookies defined

### DISCUSSION THREAD

**Discussion by siddiqkaithodu** [apify/crawlee-python#491] [Q&A] [answered]
Hello.

I cant find anything about sending requests with headers or using different methods (GET, POST). 

 I am having a big steeping curve.

Could somebody point me to the right doc

Thanks

**Comment by vdusek** [comment, marked as answer]
Hi, I suggest you to check the [Fill and submit web form](https://crawlee.dev/python/docs/examples/fill-and-submit-web-form) example in the doc.
